src/pages/TvSeries/index.jsx

```diff
@@ -133,15 +153,6 @@ export default function TVSeriesPage() {
     );
   }
 
-  if (searchQuery && searchedTVSeries.length === 0) {
```

Comment:
I see you have removed it, what is the fallback if the `searchQuery` has content, and `searchedTVSeries` has content ?

Reply:
i tried to fix not result found issue